public/static/docs/user-guide/dvcignore.md

```diff
@@ -44,9 +44,9 @@ it raises an error. Ignoring files inside such directory should be handled from
 
 The same as for [`.gitignore`](https://git-scm.com/docs/gitignore).
 
-## Example: Modification of ignored data
+## Example: Ignoring specific files
```

Comment:
I think we should either keep the previous title (it was correct), or remove it completely (if it's a single example)

Reply:
It's 1 out of 2 examples so we can either remove the subtitle or keep it. The previous one wasn't correct though, this example doesn't show any modification of previously ignored data. It's about how it looks to only ignore some files and other not.

Reply:
But yeah its a pretty basic example so if you're not convinced with the new subtitle I can just remove it for this one, that works too.

Reply:
OK I'm just removing the subtitle.